subsys/bluetooth/host/testing.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file testing.h
+ * @brief Internal API for Bluetooth testing.
+ */
+
+/*
+ * Copyright (c) 2017 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if defined(CONFIG_BT_TESTING)
```

Comment:
Could you try to keep #ifdefs out of the header file if possible. The code that calls these APIs will look like the following, and work just fine without #ifdefs in the header file:

```
if (IS_ENABLED(CONFIG_BT_TESTING) {
        bt_test_mesh_net_recv(...);
}
```

Reply:
done

Reply:
done